db/db.go

```diff
@@ -2,16 +2,31 @@ package db
 
 import (
 	"context"
+	"fmt"
 	"github.com/jackc/pgx/v4/pgxpool"
 	"os"
 )
 
-var db *pgxpool.Pool
+var (
+	db         *pgxpool.Pool
+	databaseOk = false
+)
+
+func DatabaseUsable() bool {
+	return databaseOk
```

Comment:
Why not return db != nil? We could also probably use (*Pool).Stat() to check if connection wasn't lost